src/efi/mod.rs

```diff
@@ -1115,6 +1114,8 @@ pub fn efi_exec(
     let mut ct_index = 0;
 
     // Populate with FDT table if present
+    // To boot with ACPI, we can't offer full fdt. Simply, neglect it. See https://github.com/torvalds/linux/blob/d528014517f2b0531862c02865b9d4c908019dc4/arch/arm64/kernel/acpi.c#L203
```

Comment:
Could you wrap this comment at 80 characters (or close - by putting the URL on a new line of comment.